notifications_utils/__init__.py

```diff
@@ -23,3 +24,15 @@
 # and then later remove when performing tricky formatting operations
 MAGIC_SEQUENCE = "🇬🇧🐦✉️"
 magic_sequence_regex = re.compile(MAGIC_SEQUENCE)
+
+
+def aware_utcnow():
+    return datetime.now(timezone.utc)
```

Comment:
Same question as above, but seeing this in the `notification_utils` code I see we're going to have our work cut out for us in terms of cleaning up duplication... 😆 😭